src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/valid/BEFTA_MASTER/FT_NoCCaseType/AuthorisationCaseEvent.json

```diff
@@ -40,4 +40,25 @@
   "CaseEventID" : "ApplyNoCDecision",
   "UserRole" : "caseworker-befta_master",
   "CRUD" : "CRUD"
+}, {
+  "LiveFrom" : "01/01/2017",
+  "LiveTo" : "",
+  "CaseTypeID" : "FT_NoCCaseType",
+  "CaseEventID" : "manageRespondentOrganisation",
+  "UserRole" : "caseworker-caa",
+  "CRUD" : "CRUD"
+}, {
+  "LiveFrom" : "01/01/2017",
+  "LiveTo" : "",
+  "CaseTypeID" : "FT_NoCCaseType",
+  "CaseEventID" : "manageApplicantOrganisation",
+  "UserRole" : "caseworker-caa",
+  "CRUD" : "CRUD"
+}, {
+  "LiveFrom" : "01/01/2017",
+  "LiveTo" : "",
+  "CaseTypeID" : "FT_NoCCaseType",
+  "CaseEventID" : "manageAllOrgPolicies",
+  "UserRole" : "caseworker-caa",
+  "CRUD" : "CRUD"
```

Comment:
The `caseworker-caa` role must not be granted permissions to access these new case events: as this service role must only be granted access to ONE EVENT: i.e. the nocRequest event, otherwise ACA-68 will fail.